include/openssl/opensslv.h

```diff
@@ -40,7 +40,9 @@ extern "C" {
  *  major minor fix final patch/beta)
  */
 # define OPENSSL_VERSION_NUMBER  0x10102000L
-# define OPENSSL_VERSION_TEXT    "OpenSSL 1.1.2-dev  xx XXX xxxx"
+# define OPENSSL_VERSION_NUMBER_TEXT "1.1.2-dev"
+# define OPENSSL_VERSION_TEXT "OpenSSL " OPENSSL_VERSION_NUMBER_TEXT \
+                              "  xx XXX xxxx"
```

Comment:
Small request: I think this would be more readable if the line break was made before the string, so the string composition ends up entirely on one line. Is that possible to do?

Reply:
That should be possible. I'll take a look shortly after sorting out the second CI failure. Thanks

Reply:
Maybe it would be easier (and more readable in the source) to just adjust the release-tools to generate both defines, instead of reusing the former in the latter?

Reply:
@mspncp: No.  Please.